test/test.dart

```diff
@@ -571,3 +571,16 @@ Future<void> _printPlatformInfo() async {
 
   print('Current PID $pid; OS $os, $pointerSize bit, CPU ${cpu ?? 'unknown'}');
 }
+
+class RealmA {
```

Comment:
Should we remove this now? I would suggest we move it to a method similar to `getRealm` - e.g. `getRealmAsync` rather than this custom class.